### PR TITLE
remove sudo

### PR DESCRIPTION
**Pull request by andymckay**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,8 +12,6 @@ env:
     - ACTION=loadtest
 before_script:
     - psql -c "CREATE DATABASE testdb ENCODING 'UTF8' TEMPLATE template0;" -U postgres
-before_install:
-    - sudo apt-get install postgresql-contrib
```
Comment (magopian) [inline .travis.yml]: wow, isn't that needed? Weird, can you confirm @Natim @leplatrem @ametaireau @n1k0

**Comment by magopian** [comment]
GO FASTER (travis)

File: .travis.yml (hunk 1 above)
Comment (leplatrem) [inline .travis.yml]: Indeed it was in the past, when UUIDs were generated on the DB side.

**Comment by Natim** [comment]
Thank @andymckay

**Pull request merged**